UNRELEASED.md

```diff
@@ -9,6 +9,7 @@
 - Added `ReactNode` as an accepted prop type to `primaryAction` on the `Page` component ([#3002](https://github.com/Shopify/polaris-react/pull/3002))
 - Added a `fullWidth` prop to `EmptyState` to support full width layout within a content context ([#2992](https://github.com/Shopify/polaris-react/pull/2992))
 - Added an `emptyState` prop to `ResourceList` to support in context empty states in list views ([#2569](https://github.com/Shopify/polaris-react/pull/2569))
+- Set `popOverActive` to true on keyDown in `ComboBox` to fix AutoComplete suggestions not showing when searching and selecting via keyboard ([#3028](https://github.com/Shopify/polaris-react/pull/3028))
```

Comment:
Are you referring to a prop with `popOverActive`?
I think the prop is `active` on `Popover` https://github.com/Shopify/polaris-react/blob/master/src/components/Autocomplete/components/ComboBox/ComboBox.tsx#L334

Reply:
true, i'm gonna edit this :)